### DISCUSSION TITLE
How to get JSON highlighting working in the editor()?

### DISCUSSION THREAD

**Discussion by brettshepherd** [johnlindquist/kit#702] [Q&A] [answered]
I was trying to use the `editor()` function to edit some JSON data, but I can't seem to get highlighting to work, it's all just white text. 

Here's my editor call:
` await editor({
    value: result,
    fontSize: 12,
    lineNumbers: "on",
    language: "json"
  });`

and the resulting text: 
![image](https://user-images.githubusercontent.com/31286203/151423815-711a2a85-82b5-44aa-9826-64439f6286bb.png)

compared to say, vscode: 
![image](https://user-images.githubusercontent.com/31286203/151423898-2e76600f-7300-4896-ad25-95b66f99a956.png)

Using "javascript" gets me something, but it's invalid: 
![image](https://user-images.githubusercontent.com/31286203/151424194-d8e8e821-fe5b-4c91-817e-a3bc37efbcf4.png)

Is JSON just not supported? If not is there a way to add it somehow?

**Comment by johnlindquist** [comment, marked as answer]
JSON highlighting will be supported in the next build (maybe tonight?). The `editor` will gradually improve over the coming months/year, lots of stuff to add/support.
